### ISSUE TITLE
Shadowfiend

### ISSUE THREAD

**Issue by Weqrex** [mangosR2/mangos#950]
''Creates a shadowy fiend to attack the target.  Caster receives 5% mana when the Shadowfiend attacks. Damage taken by area of effect attacks is reduced. Lasts 15 sec.''   (http://old.wowhead.com/spell=34433) (in 3.3.5a is 5% the mana that you gain when your shadowfiend hits) 

The speed of the shadowfiend is correct, it attacks every 1.5 seg, but the mana that you gain when the shadowfiend hits is incorrect. For every hit that the shadowfiend does, you gain a 10% of your mana. If it worked correctly, you should gain a 5% of your mana. 

''Has ~1.5second attack speed (10 times per 15 second life)''  (comments old.wowhead) 

So, imagine that the shadowfiend hits 10 times your target, you should gain a 50% of your mana. But now if the shadowfiend hits 10 times, you gain a 100% of your mana. 

In conclusion, the shadowfiend is giving the double mana that it should give for every hit.

(Tested in Official R2 test server)

Regards
